src/components/ClientForm/ClientForm.tsx

```diff
@@ -0,0 +1,165 @@
+"use client";
+
+import styles from "./ClientForm.module.scss";
+import { ClientFormDataDropdowns } from "../../assets/data/ClientFormData";
+import { Dropdown } from "../Dropdown/Dropdown";
+import { useState } from "react";
+import { sendContactForm } from "@/assets/apiLib/api";
+
+export function ClientForm() {
+  const [name, setName] = useState<string>("");
+  const [phone, setPhone] = useState<string>();
+  const [email, setEmail] = useState<string>();
+  const [trainingFor, setTrainingFor] = useState<string>(
+    "General Health and Fitness"
+  );
+  const [currentDaysWeek, setCurrentDaysWeek] = useState<string>("0");
+  const [futureDaysWeek, setFutureDaysWeek] = useState<string>("0");
+  const [currentFitness, setCurrentFitness] = useState<string>(
+    "1: relatively out of shape"
+  );
+  const [injuries, setInjuries] = useState<string>("No");
+  const [yesInjuries, setYesInjuries] = useState<string>();
+  const [favorite, setFavorite] = useState<string>("Cardio");
+  const [mostOften, setMostOften] = useState<string>("Cardio");
+  const [goals, setGoals] = useState<string>();
+  const [coachGoals, setCoachGoals] = useState<string>();
+
+  function handleChange(val: string, id: string) {
+    switch (id) {
+      case "trainingFor":
+        setTrainingFor(val);
+        break;
+      case "futureDaysAWeek":
+        setFutureDaysWeek(val);
+        break;
+      case "currentDaysAWeek":
+        setCurrentDaysWeek(val);
+        break;
+      case "currentFitness":
+        setCurrentFitness(val);
+        break;
+      case "injuries":
+        setInjuries(val);
+        break;
+      case "favoriteExercise":
+        setFavorite(val);
+        break;
+      case "commonExercise":
+        setMostOften(val);
+        break;
+    }
+  }
+
+  const onSubmit = async (e: any) => {
+    e.preventDefault();
+    const passingArray = [
```

Comment:
you probably want to at least validate the email and phone number before submitting but u can follow up with that